fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -311,6 +311,21 @@ public enum ErrorCode {
     ERR_TOO_MANY_ERROR_ROWS(5606, new byte[] {'2', '2', '0', '0', '0'},
             "Current error rows: %d is more than max error num: %d. Check the 'TrackingSQL' field for detailed information"),
 
+    /**
+     * 5700 - 5799: Partition
+     */
+    ERR_ADD_PARTITION_WITH_ERROR_PARTITION_TYPE(5700, new byte[] {'5', '8', '0', '3', '0'},
```

Comment:
If it's in the SQL parse stage, it's better to use 42601 as the SQLState.

Reply:
not parse stage